components/Dashboard/ItTable-Installation.vue

```diff
@@ -1,184 +1,156 @@
-      label="Location"
-      prop="location">
-    </el-table-column>
-
-<el-table-column
-   width="70px"
-      label="Phone Model"
-      prop="pmodel">
-    </el-table-column>
-
-    <el-table-column
-       width="85px"
-      label="Computer Model"
-      prop="cmodel">
-    </el-table-column>
-
-
-
-
-
-        <el-table-column
-         width="90px"
-      label="Date"
-      prop="date">
-    </el-table-column>
-
-    <el-table-column
-      width="100%"
-      label="Durum"
-     >
-       <el-tag
-           
-          disable-transitions >{{durum}}</el-tag>
-    </el-table-column>
-
-
-
-
-
-    
-    <el-table-column
-     width="266px"
-      align="left">
-      <template slot="header" >
-        <el-input
-          v-model="search"
-          size="mini"
-          placeholder="Type to search"/>
-      </template>
-      <template >
-         <el-tag
-        
-          disable-transitions @click="dialogInfo=true" >Details</el-tag>
-        <el-button
-          size="mini"
-            type="primary"
-           icon="el-icon-edit" >Edit</el-button>
-        <el-button
-          size="mini"
-          type="danger"
-         @click="dialogVisible = true" icon="el-icon-delete">Delete</el-button>
-      </template>
-    </el-table-column>
-  </el-table>
-
-
-  <el-dialog
-  title="Delete Operation"
-  :visible.sync="dialogVisible"
-  width="30%"
-  >
-  <span><b>Are you sure that you want to delete this record?</b></span>
-  <span slot="footer" class="dialog-footer">
-    <el-button @click="dialogVisible = false">Cancel</el-button>
-    <el-button type="success" @click="dialogVisible = false">Confirm</el-button>
-  </span>
-</el-dialog>
-
-  <el-dialog
-  title="Information"
-  
-  width="30%"
-  >
-  <span><b>Test Details</b></span>
-  <span slot="footer" class="dialog-footer">
-    <el-button @click="dialogInfo = false" type="success">OK</el-button>
-  </span>
-</el-dialog>
-
-
-
-</div>
+      >
+      <el-table-column align="center" label="#" type="index"></el-table-column>
+      <el-table-column
+        width="150"
+        style="height: 20%"
+        align="center"
+        label="Staff Name"
+        prop="name"
+      ></el-table-column>
+      <el-table-column width="200" align="center" label="Los" prop="los"></el-table-column>
+      <el-table-column width="100" align="center" label="Title" prop="titles"></el-table-column>
+      <el-table-column width="100" align="center" label="Location" prop="location"></el-table-column>
+      <el-table-column sortable width="150" align="center" label="Phone Model" prop="phoneModel"></el-table-column>
+      <el-table-column
+        sortable
+        width="200"
+        align="center"
+        label="Computer Model"
+        prop="computerModel"
+      ></el-table-column>
+      <el-table-column width="100" align="center" label="Date" prop="Hire_Date"></el-table-column>
+      <el-table-column width="150" align="center" label="Status" prop="process_status_short">
```

Comment:
You should pick a naming convention for your variables. I see props called `process_status_short`, `Hire_Date`, and `phoneModel ` and they all use different naming conventions.

- First of all, names should start with a lower case letter. (http://crockford.com/javascript/code.html#names)
- You can either use `camelCase` or `snake_case`, not both of them in the same project. (https://medium.freecodecamp.org/javascript-naming-conventions-dos-and-don-ts-99c0e2fdd78a)

P.S: We use `camelCase` at Rakam.

Reply:
I  changed these names as camelCase.